tasks/markdown/easy/introduction.md

```diff
@@ -5,6 +5,6 @@ Header
 - this is
 - a list
 - with bad
--formatting
+- formatting
 
 Visit our website [here] (https://forkcommitmerge.io)
```

Comment:
There are also couple other issues in this Markdown file, for example this link has extra space between the brackets and parenthesis, so it does not work, also the Header should have # character. Still, im going to merge this.